src/Cogworks.AzureSearch.Umbraco.IocExtension/Cogworks.AzureSearch.Umbraco.IocExtension.nuspec

```diff
@@ -1,29 +1,30 @@
-<?xml version="1.0"?>
+﻿<?xml version="1.0"?>
 <package>
-  <metadata>
-    <id>Cogworks.AzureSearch.Umbraco.IocExtension</id>
-    <version>$version$</version>
-    <title>Cogworks.AzureSearch.Umbraco.IocExtension</title>
-    <authors>Cogworks</authors>
-    <owners>Cogworks</owners>
-    <projectUrl>https://github.com/thecogworks/Cogworks.AzureSearch</projectUrl>
-    <repository type="git" url="https://github.com/thecogworks/Cogworks.AzureSearch" />
-    <license type="expression">Apache-2.0</license>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>A wrapper to Azure Search allowing to easily setup Azure Search indexes, searchers and using it with DI/IoC approach (currently with support for Umbraco, LightInject and Autofac).</description>
-    <copyright>The Cogworks Limited 2020</copyright>
-    <tags>Azure Search, Umbraco</tags>
-    <dependencies>
-      <group targetFramework="net48">
-        <dependency id="Cogworks.AzureSearch" version="$version$" />
-        <dependency id="Cogworks.AzureSearch.LightInject.IocExtension" version="$version$" />
-        <dependency id="UmbracoCms.Core" version="8.6.4" />
-      </group>
-    </dependencies>
-  </metadata>
-  <files>
-    <file src="bin\Release\Cogworks.AzureSearch.dll" target="lib/net48" />
-    <file src="bin\Release\Cogworks.AzureSearch.LightInject.IocExtension.dll" target="lib/net48" />
-    <file src="bin\Release\Cogworks.AzureSearch.Umbraco.IocExtension.dll" target="lib/net48" />
-  </files>
+	<metadata>
```

Comment:
Is this indenting forced by you or code style?

Reply:
this was a bug, it should be lib/net472

but spacing should be 2, but on my local it always convert to 4 - also if I change this setting. Will try again.